components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationConstants.java

```diff
@@ -78,6 +78,9 @@ private IdentityApplicationConstants(){
     public static final String ALLOW_MODIFY_USERNAME_ELEMENT = "AllowModifyUserName";
     public static final String PROMPT_CONSENT_ELEMENT = "PromptConsent";
 
+    public static final String BEGIN_CERTIFICATE = "-----BEGIN CERTIFICATE-----";
+    public static final String END_CERTIFICATE = "-----END CERTIFICATE-----";
```

Comment:
These 2 constants were defined in IdentityProvider also. Shall we use one of these?

Reply:
Fixed it